maths/perfect_number.py

```diff
@@ -21,8 +22,17 @@ def perfect(number: int) -> bool:
                 A number at most can be divisible
                 by the half of the number except
                 the number itself
-                >>> 6 at most can be divisible by 3
-                    except 6 itself
+                6 at most can be divisible by 3
+                except 6 itself
+        """
+
+        """
```

Comment:
Doctests must be in the (first) docstring of the function.
```suggestion
        """

        """
```